mytardis_rocrate_builder/rocrate_builder.py

```diff
@@ -138,13 +194,12 @@ def __add_organisation(self, organisation: Organisation) -> None:
                 "name": organisation.name,
             },
         )
-        if organisation.url:
-            org.append_to("url", organisation.url)
-        if len(organisation.identifiers) > 1:
-            for index, identifier in enumerate(organisation.identifiers):
+        self._add_optional_attr(org, "url", organisation.url)
+        if len(organisation.mt_identifiers) > 1:
```

Comment:
What about the case where the number of identifiers is exactly 1. In this case this code block would be missed and the identifier not added to the org, unless I am missing something :)

Reply:
Unless the id is the primary identifier - in this case I retract the above :)